cupy/linalg/decomposition.py

```diff
@@ -357,7 +357,16 @@ def svd(a, full_matrices=True, compute_uv=True):
         m, n = a.shape
         x = a.transpose().astype(a_dtype, order='C', copy=True)
         trans_flag = True
+
     mn = min(m, n)
+    if mn == 0:
+        if compute_uv:
+            if full_matrices:
+                return cupy.eye(m, m), cupy.array([]), cupy.empty((0, 0))
+            else:
+                return cupy.empty((m, 0)), cupy.array([]), cupy.empty((0, 0))
```

Comment:
This code path looks not tested.

Reply:
wouldn't it be directly tested by the testing.product as it's `'full_matrices': [True, False]` [here](https://github.com/cupy/cupy/blob/86eab8f0ebc124d9e6b746efa0d635d6f37062ae/tests/cupy_tests/linalg_tests/test_decomposition.py#L116) and `self.full_metrices` has been passed [here](https://github.com/cupy/cupy/blob/86eab8f0ebc124d9e6b746efa0d635d6f37062ae/tests/cupy_tests/linalg_tests/test_decomposition.py#L135)?

Reply:
I confirmed that this line is covered. Thanks!